### DISCUSSION TITLE
Buttons instead of an dropdown menu as choosable option in the config

### DISCUSSION THREAD

**Discussion by BlueSky-fur** [Sayrix/Ticket-Bot#137] [Ideas]
My idea was that it would be a pretty neat addition if we could go into the config file and change the type of "ticket openers" So instead of the classic dropdown menu we would also have buttons if the user chooses so. 

Here's an screenshot for better Understanding

![buttonsidea](https://user-images.githubusercontent.com/80676050/232376718-1e3baef7-b1fa-4030-bc93-66a5720f6c3d.png)

Every category you create in the config will then if chosen so create an interactive Button that then works just as the dropdown, you click the button and the normal Form to fill out opens up!

Would love to do it myself but I'm still in the beginning steps of learning Javascript but i thought it could be a cool little addition 😅 

Thanks in advance for reading!

**Comment by zhiyan114** [comment]
https://github.com/Sayrix/Ticket-Bot/issues/166

This feat was requested, though need more time since it require modifying the internal to also support the dropdown option. Not even sure if my skill allows it :p
